netlify.toml

```diff
@@ -0,0 +1,29 @@
+[Settings]
+  ID = "warpr"
+
+# Settings in the [build] context are global and are applied to all contexts unless otherwise overridden by more specific contexts.  
+
+[build]
+# This is the directory to change to before starting a build. 
+  base    = "."
+  # NOTE: This is where we will look for package.json/.nvmrc/etc, not root.
+# This is the directory that you are publishing from (relative to root of your repo)
+  publish = "."
+# This will be your default build command
+  command = "" 
+  
+[[redirects]]
+  from = "https://warpr.netlify.com/*"
```

Comment:
Were you able to test this somehow? I just tried and it doesn't seem to redirect:

```bash
$ http --headers "https://warpr.netlify.com/index.html"
HTTP/1.1 200 OK
Age: 23
Cache-Control: public, max-age=0, must-revalidate
Connection: keep-alive
Content-Encoding: gzip
Content-Length: 2854
Content-Type: text/html; charset=UTF-8
Date: Mon, 21 May 2018 19:06:40 GMT
Etag: "408cd03ca367ffc3ecd867265cdca797-ssl-df"
Server: Netlify
Strict-Transport-Security: max-age=31536000
Vary: Accept-Encoding
```

Reply:
We'd have to merge the PR first. I tested this on [a separate Netlify instance](https://quizzical-swirles-1abd78.netlify.com).

Reply:
Your tool looks useful... https://httpie.org